src/utils/gov.ts

```diff
@@ -0,0 +1,24 @@
+export const isWhitelisted = (url?: string) => {
+  if (!url) return false
+  if (!url.match(/https?:\/\/.*/)) {
+    url = `https://${url}`
+  }
+
+  try {
+    return new URL(url).hostname.endsWith("terra.money")
```

Comment:
This is exploitable, domains like `not-terra.money` will be considered as legit even if they are not.
I would rather do `hostname === 'terra.money' || hostname.endsWith('.terra.money')`.

Reply:
Agreed. Updated with commit [73ecce5](https://github.com/terra-money/station/pull/159/commits/73ecce57d0325d5ebfb735091cfb41cf051219dd)